test/util/numerics_test.py

```diff
@@ -234,65 +234,73 @@ def dtype(request):
     return request.param
 
 
-pad_mode_params = list(_SUPPORTED_RESIZE_PAD_MODES)
-pad_mode_params.remove('constant')
-pad_mode_params.extend(['constant 0', 'constant 1'])
-pad_mode_ids = [" pad_mode = '{}' ".format(p) for p in pad_mode_params]
+pad_modes = list(_SUPPORTED_RESIZE_PAD_MODES)
+pad_modes.remove('constant')
+pad_modes.extend([('constant', 0), ('constant', 1)])
+pad_mode_ids = [" pad_mode = '{}' {} ".format(*p)
+                if isinstance(p, tuple)
+                else " pad_mode = '{}' ".format(p)
+                for p in pad_modes]
 
 
-@pytest.fixture(scope="module", ids=pad_mode_ids, params=pad_mode_params)
+@pytest.fixture(scope="module", ids=pad_mode_ids, params=pad_modes)
 def pad_mode(request):
-    return request.param
+    if isinstance(request.param, tuple):
+        mode, const = request.param
+    else:
+        mode = request.param
+        const = 0
+
+    return mode, const
 
 
 variants = ['extend', 'restrict', 'mixed']
-setup_params = list(product(pad_mode_params, variants))
-setup_ids = [" pad_mode = '{}', variant = '{}' ".format(*p)
-             for p in setup_params]
+variant_ids = [" variant = '{}' ".format(v) for v in variants]
 
 
-@pytest.fixture(scope="module", ids=setup_ids, params=setup_params)
-def resize_setup(request):
+@pytest.fixture(scope="module", ids=variant_ids, params=variants)
+def variant(request):
+    return request.param
+
+
+@pytest.fixture(scope="module")
+def resize_setup(pad_mode, variant):
     array_in = [[1, 2, 3, 4],
                 [5, 6, 7, 8],
                 [9, 10, 11, 12]]
 
-    pad_mode, variant = request.param
-    pad_const = 0
+    mode, const = pad_mode
```

Comment:
probably should rename these to fit the parameters in the function

`pad_mode, pad_const = padding`

or something like that.